README.md

```diff
@@ -129,6 +131,20 @@ Check FAQ.md (TBA) for details.
 The original GitHub Repo is at https://github.com/EngMohamedNor/LabReportRepo
 
 
+### Steps of local test deployment
+1.download **xampp** <br>
+2.follow the website to do: 
+> https://blog.csdn.net/qing666888/article/details/81914389 <br>
```

Comment:
@dzr201732120115 

Where do you append `<br>` in the end of sentence?

**Hui**